resources/trusted_certificate.rb

```diff
@@ -28,6 +28,14 @@
     action :nothing
   end
 
+  directory certificate_path do
+    recursive true
+    mode '755'
+    owner 'root'
+    group 'staff' if platform_family?('debian')
```

Comment:
This isn't what installing the package does. It creates the directory as 

drwxr-xr-x  2 root root  4096 Oct 13 15:03 ca-certificates

on debian systems. As mentioned in the issue, I don't think this is needed if you use the recipe that installs the package.